.gitignore

```diff
@@ -52,6 +52,7 @@ sdks/python/**/*.egg
 sdks/python/LICENSE
 sdks/python/NOTICE
 sdks/python/README.md
+sdks/python/apache_beam/transforms/_external_transforms/*
```

Comment:
Maybe _generated_external_transforms, just to be very clear?

Reply:
I suggest `apache_beam/transforms/xlang/` ; then, docstrings will surface in pydoc; i think it is also fine to import from `apache_beam.transforms.xlang` directly, without having to reason which `apache_beam.io.` subpackage has the transform.

Reply:
(but importing from  `apache_beam.io` can done too )

Reply:
I think we should discourage importing from apache_beam.transforms.xlang, as this is an implementation detail that users should not have to worry about (and may change in the future).